reference/state.md

```diff
@@ -140,3 +140,17 @@ $ pulumi stack import --file my-app-production.checkpoint.json # import the new
 In addition, if you have any encrypted configuration in your stack, you'll need to re-run
 `pulumi config set --secret <key> <value>` because pulumi.com uses a different key to encrypt your secrets than the
 local endpoint.
+
+### Secrets Encryption
+
+When a secret configuration value is set (by passing `--secret` to `pulumi config set`) or created inside your program
+by using `pulumi.secret` (JavaScript) or `Output.secret` (Python) the value is encrypted with a key managed by the
+backend you are connected to.  When using the local backend, this key is derviced from a passphrase you set when
+creating your stack, and when using the web backend, it is handled by a key managed by the service.
+
+For new stacks managed with the web backend, you may choose to use the passphrase based key instead. Pass
```

Comment:
Is "web backend" the term for this?  I'm not sure what it is - but that doesn't sound right.

Reply:
It's what we use in https://pulumi.io/reference/state.html#backends, i.e. this page.  I agree this this is probably not consistent with how we talk about these things in general (I had never really heard this term before reading our docs on state management).